### PR TITLE
fix(rm_format_tags): Do not propagate deliverables format tags into HK

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR fixes:

- Removes the propagation of format tags from the deliverable file into HK
- Adds a test

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh remove_format_tags`

### How to test
- [x] login to hasta
- [x] do ´us´
1. `housekeeper delete bundle strongbison`
1. `cg workflow mip-dna store analysis /home/proj/stage/rare-disease/cases/strongbison/analysis/strongbison_config.yaml`
1. `cg upload scout strongbison`
1.  Check that the cram is loaded in IGV
1.  `cg workflow mip-dna deliver inbox strongbison`
1.  `cg upload delivery-report -f strongbison`
1. `cg workflow mip-dna store analysis /home/proj/stage/rare-disease/cases/vitalmouse/analysis/vitalmouse_config.yaml`
1. `cg upload scout vitalmouse`
1.  Check that the cram is loaded in IGV
1.  `cg workflow mip-dna deliver inbox vitalmouse`
1.  `cg upload delivery-report -f vitalmouse`

### Expected test outcome
1. `cg workflow mip-dna store analysis /home/proj/stage/rare-disease/cases/strongbison/analysis/strongbison_config.yaml`
Before:
The format tags should be added to housekeeper i.e. "meta" and "bam", should be added to HK as tags on files ✅ 
After:
All format tags from the deliverables file should be gone from HK ✅ 
1. `cg upload scout strongbison` ✅ 
1.  Check that the cram is loaded in IGV ✅ 
1.  `cg workflow mip-dna deliver inbox strongbison` ✅ 
1.  `cg upload delivery-report -f strongbison`✅ 
1. `cg workflow mip-dna store analysis /home/proj/stage/rare-disease/cases/vitalmouse/analysis/vitalmouse_config.yaml`
Only `bam-mt` and not `bam` should be present on the same file. ✅ 
No format tags should be added to HK ✅ 
1. `cg upload scout vitalmouse` ✅ 
1.  Check that the cram is loaded in IGV  ✅ 
1.  `cg workflow mip-dna deliver inbox vitalmouse`  ✅ 
1.  `cg upload delivery-report -f vitalmouse`  ✅ 

## Review
- [x] code approved by MM
- [x] tests executed by HS
- [x] "Merge and deploy" approved by MM, HS
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
Strongbison bundle in HK using master:
Screenshot 2020-08-18 at 15 29 19

Strongbison bundle using this branch:
Screenshot 2020-08-18 at 15 34 42

File: cg/meta/store/base.py [hunk 1]
```diff
@@ -105,12 +97,7 @@ def get_tags(
 ) -> list:
```
Comment (moonso) [inline cg/meta/store/base.py:97]:
```suggestion
) -> List[str]:
```

**Review by moonso — CHANGES_REQUESTED** [review]
Ok we had some discussions

File: cg/meta/store/base.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg/meta/store/base.py]: Done

**Review by moonso — APPROVED** [review]

**Comment by moonso** [comment]
Well done!

**Comment by henrikstranneheim** [comment]
@moonso Would you like to M&D?

**Comment by henrikstranneheim** [comment]
Deployed on Hasta:
Screenshot 2020-08-19 at 14 12 10

**Pull request merged**